lib/DDG/Spice/Dictionary/Definition.pm

```diff
@@ -3,6 +3,7 @@ package DDG::Spice::Dictionary::Definition;
 
 use strict;
 use DDG::Spice;
+use Text::Trim;
```

Comment:
@moollaza @MariagraziaAlastra this isn't necessary--I just forgot to remove it. :)